source/logging/StdOutLogger.cpp

```diff
@@ -31,6 +31,7 @@ void StdOutLogger::run()
         {
             writeLogMessage(std::move(message));
         }
+        this_thread::sleep_for(std::chrono::milliseconds(25));
```

Comment:
Is there a reason for choosing 25 millisecond specifically? Also Could you use a preprocessor directive constant to define this value?

Reply:
I couldn't find a best value this but I'm now trying to lower the sleep time to 1ms and it also works well but get a slight increase cpu utilization from 0.3% to 1%, basically the shorter sleep time the high cpu utilization but overall it seems all value works as long as we add sleep time and I have verified the logging functionality doesn't get impacted by this change

Reply:
I update to set constant define value for this sleep time